pkg/cli-utils.go

```diff
@@ -110,22 +110,47 @@ func NewCustomResourceDefinition(config Config) *extensionsobj.CustomResourceDef
 	return crd
 }
 
-func MarshallCrd(crd *extensionsobj.CustomResourceDefinition, outputFormat string) {
-	jsonBytes, err := json.MarshalIndent(crd, "", "    ")
+func MarshallCrd(crd *extensionsobj.CustomResourceDefinition, outputFormat string) error {
```

Comment:
the function signature is changing (return error), meaning it breaks all code importing it.

Reply:
I am ok with this, in case you folks are wondering.

Reply:
I agree, it better to have this tiny breaking change (of an unversioned lib :) ) to handle errors appropirately.